sites/partners/src/components/listings/PaperListingForm/sections/BuildingSelectionCriteria.tsx

```diff
@@ -36,7 +36,7 @@ const LotteryResults = () => {
     id: "",
     url: "",
   })
-  const cloudinaryFileService = new CloudinaryFileService(new CloudinaryFileUploader())
+  const cloudinaryFileService = new FileServiceProvider().getService()
```

Comment:
(nit, feel free to address in future CLs -- we may want to rename these variables)

Reply:
(not a nit) we should strongly type this IMO as a FileServiceInterface given that we're using a factory to make it more clear what the factory is making